### PR TITLE
Feat/chat container

### PR DESCRIPTION
**Pull request by eitanfr**

### CODE REVIEW

File: src/app/chat/channel-view/channel-view.component.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
 .channel-view-header {
-  display: flex;
-  flex-direction: row-reverse;
+  //display: flex;
```
Comment (tomermoshe) [inline src/app/chat/channel-view/channel-view.component.scss]: remove comments

File: src/app/shared/components/login-page/login-page.component.scss [hunk 2]
```diff
@@ -20,8 +20,5 @@
 }
 
 .login-logo{
-  width: 80%;
```
Comment (tomermoshe) [inline src/app/shared/components/login-page/login-page.component.scss]: return it and see

File: src/app/chat/channel-view/channel-view.component.html [hunk 3]
```diff
@@ -1,8 +1,10 @@
-<ion-header class="channel-view-header">
```
Comment (tomermoshe) [inline src/app/chat/channel-view/channel-view.component.html]: fix input when open on big screen

File: src/app/chat/channel-view/channel-view.component.scss (hunk 1 above)
Comment (eitanfr) [reply src/app/chat/channel-view/channel-view.component.scss]: ok

File: src/app/chat/channel-view/channel-view.component.html (hunk 3 above)
Comment (eitanfr) [reply src/app/chat/channel-view/channel-view.component.html]: pk

**Pull request merged**